### PR TITLE
CMS: Higgs boson to two tau leptons outreach analysis

### PR DESCRIPTION
**Pull request by stwunsch**
Second PR to include the outreach analysis of a Higgs boson decaying into two tau leptons

Related issue: #2777 
Previous PR: #2778

### CODE REVIEW

**Comment by stwunsch** [comment]
@tiborsimko Just seen that I forgot to rebase. Shall I squash the commits or you are doing it eitherway during the merge?

**Comment by stwunsch** [comment]
@tiborsimko Gentle ping :)

**Comment by tiborsimko** [comment]
> Shall I squash the commits or you are doing it eitherway during the merge?

I can squash during merge, no problem!

File: cernopendata/modules/fixtures/data/records/cms-tools-nanoaod-outreach-higgstautau.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
       {
         "checksum": "adler32:937a3ec4",
         "size": 1012110,
-        "uri": "root://eospublic.cern.ch//eos/opendata/cms/software/HiggsTauTauNanoAODOutreachAnalysis/HiggsTauTauNanoAODOutreachAnalysis-0.1.zip"
+        "uri": "root://eospublic.cern.ch//eos/opendata/cms/software/HiggsTauTauNanoAODOutreachAnalysis/HiggsTauTauNanoAODOutreachAnalysis-1.0.zip"
```
Comment (tiborsimko) [inline cernopendata/modules/fixtures/data/records/cms-tools-nanoaod-outreach-higgstautau.json]: I'll grab the new tarball and I see that `skim.cxx` and `histograms.py` have also changed so I re-download these to EOS and update checksums etc accordingly.

Comment (stwunsch) [reply cernopendata/modules/fixtures/data/records/cms-tools-nanoaod-outreach-higgstautau.json]: Perfect!

**Pull request merged**